rest/handler.go

```diff
@@ -347,13 +351,15 @@ func (h *handler) checkAuth(context *db.DatabaseContext) (err error) {
 		}
 	}
 
+	realm := ` realm="` + base.ProductNameString + `"`
```

Comment:
Concatenating half of the WWW-Authenticate header string here, and the other half below, seems unnecessary.  Any reason to not just define this as "Basic realm=" + base.ProductNameString + `"` here?